Static IP addresses (#3803)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/about-cloud/regions-ip-addresses.md

[^1]: These regions support [multi-tenant](/docs/cloud/about-cloud/tenancy) deployment environments hosted by dbt Labs.

## Static IP addresses

> dbt Cloud, like many cloud services, relies on underlying AWS cloud infrastructure for operations. Due to this, while we can offer exact URLs for access, we're unable to provide a list of static IP addresses to configure connections.

Reviewer: ```suggestion
dbt Cloud, like many cloud services, relies on underlying AWS cloud infrastructure for operations. While we can offer exact URLs for access, we're unable to provide a list of static IP addresses to configure connections due to its cloud nature.
```

> 1. Dynamic IP Addresses &mdash; Our cloud infrastructure is built upon Amazon Web Services (AWS). Due to the adaptive nature of cloud services, while dbt Cloud can provide exact URLs for streamlined access, the underlying IP addresses are dynamic and will change occasionally. AWS manages the IP ranges and may change them according to their operational requirements.

Reviewer: ```suggestion
1. Dynamic IP addresses &mdash; Our cloud infrastructure uses Amazon Web Services (AWS). dbt Cloud offers exact URLs for streamlined access, but the dynamic nature of cloud services means the underlying IP addresses change occasionally. This is because AWS manages the IP ranges and may change them according to their operational needs.
```

> 2. Embracing Hostnames for Consistent Access &mdash; To ensure uninterrupted access to dbt Cloud services, we recommend using hostnames for configurations. Hostnames provide a stable reference point that remains consistent, regardless of any changes in underlying IP addresses. This is an industry-standard pattern that's also employed by organizations such as Snowflake.

Reviewer: sugg sentence case

```suggestion
2. Embracing hostnames for consistent access &mdash; To ensure uninterrupted access to dbt Cloud services, we recommend using hostnames for configurations. Hostnames provide a stable reference point that remains consistent, regardless of any changes in underlying IP addresses. This is an industry-standard pattern that's also employed by organizations such as Snowflake.
```

> 3. Optimizing VPN Connections &mdash; Customers who leverage VPN connections, we suggest integrating a proxy alongside the VPN. This strategy allows you to maintain consistent IP addresses for your connections, facilitating seamless traffic flow through the VPN and onward to dbt Cloud. By employing a proxy and a VPN, traffic can be directed through the VPN and then to dbt Cloud. If there's any need to integrate with additional services, it's crucial to set up the proxy accordingly.

Reviewer: ```suggestion
3. Optimizing VPN connections &mdash; We suggest integrating a proxy alongside the VPN for users who leverage VPN connections. This strategy enables steady IP addresses for your connections, facilitating smooth traffic flow through the VPN and onward to dbt Cloud. By employing a proxy and a VPN, traffic can be directed through the VPN and then to dbt Cloud. If there's any need to integrate with additional services, it's crucial to set up the proxy accordingly.
```

There are two ways to view your dbt Cloud IP addresses:
- If no projects exist in the account, create a new project, and the IP addresses will be displayed during the **Configure your environment** steps.
- If you have an existing project, navigate to **Account Settings** and ensure you are in the **Projects** pane. Click on a project name, and the **Project Settings** window will open. Locate the **Connection** field and click on the name. Scroll down to the **Settings**, and the first text block lists your IP addresses. 

### Static IP addresses

> dbt Cloud, like many cloud services, relies on underlying AWS cloud infrastructure for operations. While we can offer exact URLs for access, we're unable to provide a list of static IP addresses to configure connections due to its cloud nature.

Author: ```suggestion
dbt Cloud, like many cloud services, relies on underlying AWS cloud infrastructure for operations. While we can offer static URLs for access, we cannot provide a list of IP addresses to configure connections due to the nature of AWS cloud services.
```

> 1. Dynamic IP addresses &mdash; Our cloud infrastructure uses Amazon Web Services (AWS). dbt Cloud offers exact URLs for streamlined access, but the dynamic nature of cloud services means the underlying IP addresses change occasionally. This is because AWS manages the IP ranges and may change them according to their operational needs.

Author: ```suggestion
1. Dynamic IP addresses &mdash; dbt Cloud infrastructure uses Amazon Web Services (AWS). dbt Cloud offers static URLs for streamlined access, but the dynamic nature of cloud services means the underlying IP addresses change occasionally. AWS manages the IP ranges and may change them according to their operational and security needs.
```

> 1. Dynamic IP addresses &mdash; dbt Cloud infrastructure uses Amazon Web Services (AWS). dbt Cloud offers static URLs for streamlined access, but the dynamic nature of cloud services means the underlying IP addresses change occasionally. AWS manages the IP ranges and may change them according to their operational and security needs.

Reviewer 2: Can we change these to bullets?

> 1. Dynamic IP addresses &mdash; dbt Cloud infrastructure uses Amazon Web Services (AWS). dbt Cloud offers static URLs for streamlined access, but the dynamic nature of cloud services means the underlying IP addresses change occasionally. AWS manages the IP ranges and may change them according to their operational and security needs.

Author: ```suggestion
* Dynamic IP addresses &mdash; dbt Cloud infrastructure uses Amazon Web Services (AWS). dbt Cloud offers static URLs for streamlined access, but the dynamic nature of cloud services means the underlying IP addresses change occasionally. AWS manages the IP ranges and may change them according to their operational and security needs.
```

> 2. Using hostnames for consistent access &mdash; To ensure uninterrupted access to dbt Cloud services, it's recommended that you use hostnames for configurations. Hostnames provide a consistent reference point, regardless of any changes in underlying IP addresses. This is an industry-standard practice employed by organizations such as Snowflake.

Reviewer 2: ```suggestion
* Using hostnames for consistent access &mdash; To ensure uninterrupted access, we recommend that you dbt Cloud services using hostnames. Hostnames provide a consistent reference point, regardless of any changes in underlying IP addresses. We are aligning with an industry-standard practice employed by organizations such as Snowflake.
```

> 2. Embracing hostnames for consistent access &mdash; To ensure uninterrupted access to dbt Cloud services, we recommend using hostnames for configurations. Hostnames provide a stable reference point that remains consistent, regardless of any changes in underlying IP addresses. This is an industry-standard pattern that's also employed by organizations such as Snowflake.

Author: ```suggestion
2. Using hostnames for consistent access &mdash; To ensure uninterrupted access to dbt Cloud services, it's recommended that you use hostnames for configurations. Hostnames provide a consistent reference point, regardless of any changes in underlying IP addresses. This is an industry-standard practice employed by organizations such as Snowflake.
```

> 3. Optimizing VPN connections &mdash: You should integrate a proxy alongside VPN for users who leverage VPN connections. This strategy enables steady IP addresses for your connections, facilitating smooth traffic flow through the VPN and onward to dbt Cloud. By employing a proxy and a VPN, traffic can be directed through the VPN and then to dbt Cloud. If there's any need to integrate with additional services, setting up the proxy accordingly is crucial.

Reviewer 2: ```suggestion
* Optimizing VPN connections &mdash: You should integrate a proxy alongside VPN for users who leverage VPN connections. This strategy enables steady IP addresses for your connections, facilitating smooth traffic flow through the VPN and onward to dbt Cloud. By employing a proxy and a VPN, you can direct traffic through the VPN and then to dbt Cloud. It's crucial to set up the proxy if you need to integrate with additional services.
```

> 3. Optimizing VPN connections &mdash; We suggest integrating a proxy alongside the VPN for users who leverage VPN connections. This strategy enables steady IP addresses for your connections, facilitating smooth traffic flow through the VPN and onward to dbt Cloud. By employing a proxy and a VPN, traffic can be directed through the VPN and then to dbt Cloud. If there's any need to integrate with additional services, it's crucial to set up the proxy accordingly.

Author: ```suggestion
3. Optimizing VPN connections &mdash: You should integrate a proxy alongside VPN for users who leverage VPN connections. This strategy enables steady IP addresses for your connections, facilitating smooth traffic flow through the VPN and onward to dbt Cloud. By employing a proxy and a VPN, traffic can be directed through the VPN and then to dbt Cloud. If there's any need to integrate with additional services, setting up the proxy accordingly is crucial.
```
